### PR TITLE
Update Blip “figma”

### PR DESCRIPTION
**Pull request by darcey007**
Automatically generated by Decap CMS

### CODE REVIEW

File: src/content/blips/figma.md [hunk 1]
```diff
@@ -24,4 +32,15 @@ projectIds:
   - diana-health
   - capable-health
   - cumulus
----
```
Comment (julienblin) [inline src/content/blips/figma.md:27]: For this in particular, I wonder if there is value in linking it to projects as most of them will use it anyway, and maintaining that list coud be tedious.
We could simply say in the opinion that its used across the board on almost all our projects, and leave it empty.
Thoughts, anyone?

File: src/content/blips/figma.md [hunk 2]
```diff
@@ -1,19 +1,27 @@
...
+  * Figma for VS Code extension allows devs to get code suggestions, inspect designs and post comments from their IDE: <https://marketplace.visualstudio.com/items?itemName=figma.figma-vscode-extension>
+
+
 
   It has the following weaknesses:
 
-  - The way resources are organized
 
-link: 
+  * The way resources are organized. Currently, ACN designers use the first page to post designs that are Ready For Dev. All other pages are for work in progress.
```
Comment (julienblin) [inline src/content/blips/figma.md:23]: Are there any methodologies / organizations technique resources we could point to?

**Review by julienblin — APPROVED** [review]
I believe these are worthy additions.
I just wonder about the project links. Curious to hear other's opinions.

File: src/content/blips/figma.md [hunk 3]
```diff
@@ -1,19 +1,27 @@
...
+  experiences. It is the current best-in-class tool for product designers.
 opinion: >-
   It has the following strengths:
-  
-  - Its version control of design artifacts
-  - Easy to learn and excellent documentation and tutorials
-  - Figjam whiteboarding tool facilitates team collaboration around figma artifacts
+
+
+  * Its version control of design artifacts - Easy to learn and excellent documentation and tutorials -
```
Comment (annaelleduff) [inline src/content/blips/figma.md:10]: it should be said that this is on a paid plan solely

File: src/content/blips/figma.md [hunk 4]
```diff
@@ -1,19 +1,27 @@
...
   It has the following strengths:
-  
-  - Its version control of design artifacts
-  - Easy to learn and excellent documentation and tutorials
-  - Figjam whiteboarding tool facilitates team collaboration around figma artifacts
+
+
+  * Its version control of design artifacts - Easy to learn and excellent documentation and tutorials - 
+
+  * FigJam whiteboarding tool (included with Figma) facilitates team collaboration around Figma artifacts. However, FigJam is not as popular as Miro for collaboration between Product and Engineering as it has fewer features.
```
Comment (annaelleduff) [inline src/content/blips/figma.md:12]: to complete the comparison I think it should be noted that figjam is cheaper than miro (3$ vs 8$ for the first paid plan that allows more than 3 boards)

File: src/content/blips/figma.md (hunk 1 above)
Comment (annaelleduff) [reply src/content/blips/figma.md:27]: could make sense, but then if one project uses something else, how would we make it obvious? would we even want to make it obvious?

Comment (julienblin) [reply src/content/blips/figma.md:27]: @annaelleduff - I am not sure it really matters.
The idea behind the project links is for people to find others who have been working on it, so they can ask further questions. For figma it feels kinda obvious that you would reach out to the design team.

**Review by jorge-vasquez-acn — APPROVED** [review]

**Pull request merged**